src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,35 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) injector.getInstance(AuthenticationService.class);
+        try {
+            authenticationService.register("katedeshchynska@gmail.com", "12345678");
+            System.out.println(authenticationService
+                    .login("katedeshchynska@gmail.com", "12345678"));
+        } catch (AuthenticationException e) {
+            throw new RuntimeException(e);
+        }
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        User user = userService.findByEmail("katedeshchynska@gmail.com").get();
```

Comment:
AuthenticationService.login() return User. Maybe be better to use AuthenticationService to find a user instead of UserService